components/validator/guardian.go

```diff
@@ -199,10 +190,10 @@ func (g *Guardian) checkConfirmCondition(ctx context.Context, transactionId *big
 }
 
 func (g *Guardian) ValidateL2Output(ctx context.Context, outputRoot eth.Bytes32, l2BlockNumber uint64) (bool, error) {
```

Comment:
It looks like u've removed all context arguments from the Guardian receiver, can we do something similar here?

Reply:
I left out all context arguments of the functions used in the test packages because test context needs to be injected.